extension/src/popup/components/BottomNav/index.tsx

```diff
@@ -1,17 +1,40 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES } from "popup/constants/routes";
 
 import HistoryIcon from "popup/assets/icon-history.svg";
-import SwapIcon from "popup/assets/icon-swap.svg";
 import WalletIcon from "popup/assets/icon-wallet.svg";
 import SettingsIcon from "popup/assets/icon-settings.svg";
 
 import "./styles.scss";
 
+interface NavLinkProps {
+  to: string;
+  children: React.ReactNode;
+}
+
+const BottomNavLink = ({ children, to }: NavLinkProps) => (
+  <NavLink
+    to={to}
+    activeClassName="BottomNav--link--active"
```

Comment:
having distinct routes for Account, History, and Settings allows us to highlight the active tab automatically